src/common/bench_interrupt_latency_test.c

```diff
@@ -57,6 +57,7 @@ __weak void bench_exit_timer_isr(void)
  */
```

Comment:
For the commit message: you give a more detailed description of the issue on the PR message - could you please add that info to the commit message? It's better to have more information in it.